doc/api/n-api.md

```diff
@@ -4537,6 +4594,7 @@ This API may only be called from the main thread.
 [Working with JavaScript Values]: #n_api_working_with_javascript_values
 [Working with JavaScript Values - Abstract Operations]: #n_api_working_with_javascript_values_abstract_operations
 
+[`napi_add_finalizer`]: #n_api_napi_add_finalizer
```

Comment:
`` [`napi_add_finalizer`] `` -> `` [`napi_add_finalizer()`] `` (according with used references above).

Reply:
@vsemozhetbyt I changed it by removing the brackets where I use the reference.